opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -63,20 +63,23 @@
 .. versionadded:: 0.1.0
 .. versionchanged:: 0.3.0
     `TracerProvider` was introduced and the global ``tracer`` getter was replaced
-    by `tracer_provider`.
+    by `get_tracer_provider`.
 """
 
 import abc
 import enum
-import logging
 import types as python_types
 import typing
 from contextlib import contextmanager
+from logging import getLogger
 
+from pkg_resources import iter_entry_points
+
+from opentelemetry.configuration import Configuration
 from opentelemetry.trace.status import Status
-from opentelemetry.util import loader, types
+from opentelemetry.util import types
 
-logger = logging.getLogger(__name__)
+_LOGGER = getLogger(__name__)
```

Comment:
this is inconsistent with the rest of the codebase

Reply:
Fixed